singlem/lyrebird.py

```diff
@@ -0,0 +1,455 @@
+#!/usr/bin/env python3
+
+__author__ = "Ben Woodcroft"
+__copyright__ = "Copyright 2015-2024"
+__credits__ = ["Ben Woodcroft", "Samuel Aroney", "Rossen Zhao"]
+__license__ = "GPL3+"
+__maintainer__ = "Ben Woodcroft"
+__email__ = "benjwoodcroft near gmail.com"
+__status__ = "Development"
+
+import logging
+import sys
+import os
+import gzip
+import tempfile
+import json
+
+from bird_tool_utils import *
+from bird_tool_utils.people import *
+
+sys.path = [os.path.join(os.path.dirname(os.path.realpath(__file__)),'..')] + sys.path
+
+import singlem
+import singlem.pipe as pipe
+from singlem.pipe import SearchPipe
+from singlem.condense import Condenser
+from singlem.metapackage import DATA_ENVIRONMENT_VARIABLE, CUSTOM_TAXONOMY_DATABASE_NAME
+from singlem import OTU_TABLE_OUTPUT_FORMAT, ARCHIVE_TABLE_OUTPUT_FORMAT
+
+from singlem.condense import DEFAULT_MIN_TAXON_COVERAGE as CONDENSE_DEFAULT_MIN_TAXON_COVERAGE
+from singlem.condense import DEFAULT_GENOME_MIN_TAXON_COVERAGE as CONDENSE_DEFAULT_GENOME_MIN_TAXON_COVERAGE
+from singlem.condense import DEFAULT_TRIM_PERCENT as CONDENSE_DEFAULT_TRIM_PERCENT
+
+DEFAULT_WINDOW_SIZE = 60
```

Comment:
remove these?

Reply:
This would need the same imports in main.py to be moved into the functions for condense args and get_min_taxon_coverage.